auth-web/src/components/auth/staff/account-management/StaffAccountManagement.vue

```diff
@@ -178,8 +182,11 @@ export default class StaffAccountManagement extends Vue {
     await this.getCodes()
     await this.syncPendingStaffOrgs()
     await this.syncRejectedStaffOrgs()
-    await this.syncPendingInvitationOrgs()
+    await this.getCodes()
```

Comment:
Please remove this line as await this.getCodes() as we have it in 182

Reply:
Removed. Thanks.